From d2fabc31efd91f9ac5ae8721d48367fee036c627 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Wed, 18 Jan 2023 10:55:52 +0100
Subject: [PATCH 184/464] arm64: dts: rockchip: rk3399-pinebook-pro: Fix VDO /
 display output

This fixes Alt-DP mode on Pinebook Pro.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
index 7d7820f26543..b3c985dd32a3 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
@@ -723,7 +723,7 @@ connector {
 			altmodes {
 				dp {
 					svid = <0xff01>;
-					vdo = <0x0c0046>;
+					vdo = <0x0c46>;
 				};
 			};
 
-- 
2.34.1

